### PR TITLE
Unify C# naming notes and comments

### LINKED ISSUE #9048: Confusing part in the step by step guide
**Your Godot version:**
4.2.1
**Issue description:**
The tutorial has the function _on_button_pressed() in C# as OnButtonPressed() but that doesn't work in the engine. It's in the part where they talk about stopping the _Process() function.
**URL to the documentation page:**
https://docs.godotengine.org/en/stable/getting_started/step_by_step/signals.html

### PR DESCRIPTION
**Pull request by skyace65**
Doing this because of discussion in #9048.

Currently there's a note in the first 2D game tutorial about naming, the first 3D game has comments in all of its code for signals, and the signals page in step by step has nothing. This PR makes it so all sections have a note and code comments. The notes are being used in addition to comments since comments can't be translated. Technically we could have the note just in the signals page, since you're supposed to read that before either of the first game tutorials, but I don't think we should assume people won't skim or skip around. Plus if someone's going through these pages over multiple days and not in one day it's good to have reminders.

Closes #9048.

### CODE REVIEW

File: getting_started/first_2d_game/03.coding_the_player.rst [hunk 1]
```diff
@@ -407,6 +407,7 @@ Next, add this code to the function:
 
  .. code-tab:: csharp
 
+    // We also specified this function name in PascalCase in the editor's connection window
```
Comment (AThousandShips) [inline getting_started/first_2d_game/03.coding_the_player.rst:410]:
```suggestion
    // We also specified this function name in PascalCase in the editor's connection window.
```
For all the comments

File: getting_started/step_by_step/signals.rst [hunk 2]
```diff
@@ -326,6 +339,7 @@ bottom of our script and use it to toggle our sprite's visibility.
 
  .. code-tab:: csharp C#
 
+    // We also specified this function name in PascalCase in the editor's connection window
```
Comment (AThousandShips) [inline getting_started/step_by_step/signals.rst:342]:
```suggestion
```
This isn't connected by the editor

File: getting_started/step_by_step/signals.rst [hunk 3]
```diff
@@ -393,11 +407,13 @@ Here is the complete ``sprite_2d.gd`` file for reference.
             Position += velocity * (float)delta;
         }
 
+        // We also specified this function name in PascalCase in the editor's connection window
         private void OnButtonPressed()
         {
             SetProcess(!IsProcessing());
         }
 
+        // We also specified this function name in PascalCase in the editor's connection window
```
Comment (AThousandShips) [inline getting_started/step_by_step/signals.rst:416]:
```suggestion
```

**Review by AThousandShips — APPROVED** [review]

**Comment by skyace65** [comment]
Fixed. I've also added a period to the code comments in the first 2D game that I was copying from.

**Review by raulsntos — APPROVED** [review]
LGTM. Thanks for taking care of this :+1:

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! 🎉

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2 in https://github.com/godotengine/godot-docs/pull/9648.

**Pull request merged**